src/Jackalope/Query/Row.php

```diff
@@ -107,7 +107,7 @@ public function __construct(FactoryInterface $factory, ObjectManager $objectMana
                 $this->path[$selectorName] = $column['dcr:value'];
             } else {
                 if ('jcr:primaryType' === substr($column['dcr:name'], -15)) {
-                    $this->defaultSelectorName = $selectorName;
+                    $this->defaultSelectorName = $column['dcr:value'];
```

Comment:
is this change related to the fix?

Reply:
No. I actually wanted to make two pull requests of my two commits, but I lost the fight with GitHub. This is the commit of my "other pull request" (see https://github.com/jackalope/jackalope/pull/351/commits/018f1968c77d0a7e099b27d301122f87534e645b). I have no clue how to fix my mistake.

Reply:
you can checkout your fork, create a new branch from master and cherry-pick that second commit `git cherry-pick 018f196`. then go to this branch and `git rebase -i origin/master` and remove the second commit. that will leave only the first commit in the branch. then `git push --force-with-lease` to push the change upstream.

Reply:
this line does currently break our build in sulu/sulu on jackrabbit `PHPCR\RepositoryException: Attempting to get the path for a non existent selector: nt:unstructured`. doctrine dbal seems to work fine. any ideas?

/cc @dbu @middlebrain

Reply:
Running the test in jackalope-jackrabbit errors with:

```
1) PHPCR\Tests\Query\QuerySql2OperationsTest::testQueryFieldDate
PHPCR\RepositoryException: Attempting to get the path for a non existent selector: nt:unstructured

/private/tmp/jackalope-jackrabbit/vendor/jackalope/jackalope/src/Jackalope/Query/Row.php:208
/private/tmp/jackalope-jackrabbit/vendor/phpcr/phpcr-api-tests/tests/Query/QuerySql2OperationsTest.php:70

ERRORS!
Tests: 1866, Assertions: 8624, Errors: 1, Skipped: 62, Incomplete: 12.
```